### PR TITLE
Fill out entry for sparkle and add example

### PR DESCRIPTION
**Pull request by kfrn**
Now with more Chow Yun-Fat :)

> I gotta get up in the morning and it's sparkle, Neely, sparkle!

### CODE REVIEW

**Comment by retokromer** [comment]
In this one, I guess you should be clearer in saying that, if the whites on a positive are photographically reproduced dirt from a negative, then that the **_negative_** has to be cleaned and not the positive.

How are the small white spots physically on the positive image, that are due to missing emulsion, called in English? Those risk to **_increase_** by inappropriate cleaning. Both phenomena looks quite similar and may easily be confused by unskilled persons. A high magnifying glass or a microscope is needed to distinguish the two.

File: _artifacts/sparkle.md [hunk 1]
```diff
@@ -6,12 +6,16 @@ tags: [Analog, Film]
 published: true
 ---
 
-[Definition to go here!]
+Sparkle refers to small white spots on the image, usually due to tiny pieces of dust or dirt being present on the film negative (or internegative). Sparkle often occurs more frequently at the beginning or end of a reel of film.
 
 ## Can it be fixed?
 
-[wanted!]
+Sparkle can be reduced by cleaning film using an ultrasonic cleaner, particle transfer rollers, or even hand-cleaning with a velvet. Rewashing the film will also help reduce sparkle.
```
Comment (retokromer) [inline _artifacts/sparkle.md]: clean the negative if white spot in positive

**Comment by kfrn** [comment]
Good point, I shall make an update.

I'd call that 'flaking emulsion' or 'emulsion lift', is that what you mean? What's the German/French term? It's not that common in my experience, but definitely a good thing to note.

**Comment by retokromer** [comment]
I think is what you say. I will ask Dominic Case and/or Paul Collard.

This is quite frequent in badly processed prints (too fast, temperature too high, old chemicals, not enough washing, not perfectly clean water, etc.). I have seen tons of them in the former Eastern Bloc, especially on Orwo material, as well as in many Asian countries, e.g. Indonesia or the Philippines.

**Comment by kfrn** [comment]
In fact, I checked Dominic Case's book to see if I could find a standard term. :)

Good to know!

**Comment by retokromer** [comment]
BTW: The 2<sup>nd</sup> edition of Dominic’s book has also been well translated in German.

**Comment by kfrn** [comment]
It's the second edition I own, too.  
I really like all of those Focal Press film technology books.

**Comment by retokromer** [comment]
I have both the editions, and do actually use both. The 1<sup>st</sup> focusses on classic analogue in more detail, while the 2<sup>nd</sup> includes the mixed times (analogue shooting, digital post-production and analogue screening).

I fully agree regarding Focal! In my opinion _The Filmmaker’s Guide to Digital Imaging_ by Blain Brown is very useful for audio-visual archivists as well.

**Comment by kfrn** [comment]
Haven't read that one, I'll look out for it! I do however have one of their books from the 70s - _Your Film and the Lab_ - which is great for photochemical workflows.

**Comment by retokromer** [comment]
I’m not sure I know that one, thank you for the hint! Dominic‘s 1<sup>st</sup> is titled _Motion Picture Film Processing_.

**Pull request merged**